src/frontend/tsx/Chessboard.tsx

```diff
@@ -77,10 +77,21 @@ export function Chessboard () {
   const initBoard = () => {
     setApi(Chessground(boardRef.current, CHESSGROUND.baseConfig))
   }
+  
```

Comment:
When I compiled with Docker it flagged up a "trailing spaces" error here; just clear the line so it's actually empty.